boards/shields/amg88xx/doc/index.rst

```diff
@@ -1,71 +1,137 @@
 .. _amg88xx_shields:
 
-Panasonic Grid-EYE (AMG88xx) Shields
-####################################
+Panasonic Grid-EYE Shields
+##########################
 
 Overview
 ********
 
-The Panasonic AMG88XX is an 8x8 (64) pixel infrared array sensor available
-only in a SMD design. Here are two sources for easily working with this sensor:
+The Panasonic Grid-EYE is an infra-red sensor, which is comprised of 64 individual thermopile elements arranged in an 8x8 matrix.
+
+The sensor comes in a compact surface mount package and can be very easily integrated into thermometry applications.
+
+Grid-EYE is available in different varieties which differ in field of view, amplification factor and operating voltages. For example AMG8833 features a 60 degree field of view, a high gain amplification factor and 3.3V operating voltage.
```

Comment:
This is hard to review or read in text editor, please keep the line length as in original.

Reply:
Fixed in [b040a75](https://github.com/zephyrproject-rtos/zephyr/pull/53244/commits/b040a75cf41d69878d2e601fda12d6fb097f6aef)